packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -648,6 +683,8 @@ class InspectorController implements InspectorServiceClient {
     }
 
     animateTo(selectedNode);
+
+    notifySelectionListeners();
```

Comment:
this might be the only place we need to notify that selection changed. I'm not sure though. To test, change the selection on the device in the inspector and see if the value changed.

Reply:
you are right, this is the only place to notify that selection has changed. thanks jacob!